service/hook/bitbucketserver/bitbucketserver.go

```diff
@@ -270,6 +270,15 @@ func (hp HookProvider) TransformRequest(r *http.Request) hookCommon.TransformRes
 			}
 		}
 
+		return transformPullRequestEvent(pullRequestEvent)
+	} else if eventKey == "pr:merged" {
```

Comment:
All the `else if`s in this conditional seem to be doing the same, am I missing something here? Pls consider DRYing it up.